systemd/cloud-init-local.service.tmpl

```diff
@@ -1,6 +1,7 @@
 ## template:jinja
 [Unit]
-Description=Initial cloud-init job (pre-networking)
+# https://cloudinit.readthedocs.io/en/latest/explanation/boot.html
+Description=Cloud-init: Local Stage
```

Comment:
I really want to keep `(pre-networking)` somewhere in there as Local turns into something generic that is not understood unless someone actually opens the uint file, sees the commented link and reads about the different stages. pre-network also implies a more clear ordering that is easier to decipher.
WDYT?
```suggestion
Description=Cloud-init: Local Stage (pre-network)
```